tensorflow-framework/src/main/java/org/tensorflow/framework/activations/AbstractActivation.java

```diff
@@ -14,18 +14,44 @@
 =======================================================================*/
 package org.tensorflow.framework.activations;
 
+import java.util.Collections;
+import java.util.Map;
 import org.tensorflow.op.Ops;
-import org.tensorflow.types.family.TNumber;
 
 /** Abstract base class for Activations */
-public abstract class AbstractActivation<T extends TNumber> implements Activation<T> {
+public abstract class AbstractActivation implements Activation {
 
   /** The TensorFlow Ops */
   protected Ops tf;
 
   /** Creates the abstract class for an AbstractActivation */
   protected AbstractActivation() {}
 
+  /**
+   * Gets a configuration map, this default implementation returns a singleton Map, with {@code
+   * "name"} as the key, and the @code name} parameter as its value;
+   *
+   * @param name the name of the Activation as known by TensorFlow engine.
+   * @return the configuration map
+   */
+  protected Map<String, Object> getConfig(String name) {
```

Comment:
Should this be no-args and call `getName()` to get the name?

Reply:
`getConfig(name)` is a helper that creates a singleton Map with a "name" key.
while` getConfig() `is an abstract method that may create a more complex config Map.
Perhaps change to `getDefaultConfig` and populate it using `getName()`.